### PR TITLE
Added direct Lambda invoke documentation

### LINKED ISSUE #745: Add direct invoke docs for Lambda
Add instructions to invoke Lambda directly
```
$(AWS_CLI) lambda invoke --function-name localstack-lambda-url-example \
	--cli-binary-format raw-in-base64-out \
	--payload '{"body": "{\"num1\": \"10\", \"num2\": \"10\"}" }' output.txt
```

### PR DESCRIPTION
**Pull request by cabeaulac**
The documentation did not have example of invoking Lambda directly. I added it.

Closes #745

### CODE REVIEW

**Comment by tinyg210** [comment]
hey @cabeaulac, I tried this out and I have some issues with `awslocal`, I get an error message: `Unknown options: --cli-binary-format, output.txt`. 
My `aws --version` outputs `aws-cli/2.13.6 Python/3.11.4 Darwin/22.5.0 exe/x86_64 prompt/off`. 
On the other hand if i use 
```
aws --endpoint "http://localhost:4566" lambda invoke --function-name localstack-lambda-url-example \
                                              --cli-binary-format raw-in-base64-out \
                                              --payload '{"body": "{\"num1\": \"10\", \"num2\": \"10\"}" }' \
                                               output.txt
```
it works as expected 🤔. Was wondering if you tested using `--endpoint` or if I'm missing something.

**Comment by joe4dev** [comment]
Direct invoke is easier because it does not require manually updating the HTTP request URL (I find that annoying for demo purposes). The main question is where it should go (prominent at the top or lower down the page). Any preference @HarshCasper ?

Depending on the version of the AWS CLI, a direct invoke requires different options (which makes this a bit annoying):
> The cli-binary-format option is required if you're using AWS CLI version 2.
Source: https://docs.aws.amazon.com/cli/latest/reference/lambda/invoke.html#examples

**Comment by tinyg210** [comment]
Yes, that's why I thought it was weird, cause I am using v2 and it results in `Unknown options: --cli-binary-format`, when using `awslocal`. Is it possible that `awslocal` does not support that option?
Regardless, we should mention that in the docs.

**Comment by MarcelStranak** [comment]
@tinyg210 Perhaps the [documentation page](https://docs.localstack.cloud/user-guide/integrations/aws-cli/#localstack-aws-cli-awslocal) for the AWS CLI local does not explain it very well?

```
$ aws --version
aws-cli/2.13.6 Python/3.11.4 Windows/10 exe/AMD64 prompt/off

$ awslocal --version
aws-cli/1.29.2 Python/3.10.8 Windows/10 botocore/1.31.2
```

**Comment by tinyg210** [comment]
Got it. Didn't check the `awslocal --version` and assumed it would directly use the default `aws-cli`. Sorry for the confusion

**Comment by cabeaulac** [comment]
> Got it. Didn't check the `awslocal --version` and assumed it would directly use the default `aws-cli`. Sorry for the confusion

Glad this got clarified. I will update the docs now and say it requires AWS CLI v2. And provide --endpoint example. And provide `localstack` AWS profile config example.

**Comment by cabeaulac** [comment]
Ok. I update PR with example using endpoint url and link to sample repo.

File: content/en/user-guide/aws/lambda/index.md [hunk 1]
```diff
@@ -67,13 +67,35 @@ In the old Lambda provider, you could create a function with any arbitrary strin
 ### Invoke the Function with AWS CLI
 Invoke the function directly. See ['Lambda CLI invoke docs'](https://docs.aws.amazon.com/cli/latest/reference/lambda/invoke.html) 
 The command below encodes the body as a JSON string like an event coming from API Gateway or Lambda URL does.
+This requires AWS CLI v2. To see examples of using the 3 ways to use the AWS CLI with LocalStack, see this repo. [Simple AWS CLI Sample with Lambda](https://github.com/localstack-samples/sample-simple-invoke-lambda)
```
Comment (tinyg210) [inline content/en/user-guide/aws/lambda/index.md:70]: Nit:
```suggestion
This requires AWS CLI v2. To see examples of the three ways to use the AWS CLI with LocalStack, see this repo: [Simple AWS CLI Sample with Lambda](https://github.com/localstack-samples/sample-simple-invoke-lambda)
```

File: content/en/user-guide/aws/lambda/index.md [hunk 2]
```diff
@@ -64,6 +64,38 @@ $ awslocal lambda create-function \
 In the old Lambda provider, you could create a function with any arbitrary string as the role, such as `r1`. However, the new provider requires the role ARN to be in the format `arn:aws:iam::000000000000:role/lambda-role` and validates it using an appropriate regex. However, it currently does not check whether the role exists.
 {{< /alert >}}
 
+### Invoke the Function with AWS CLI
+Invoke the function directly. See ['Lambda CLI invoke docs'](https://docs.aws.amazon.com/cli/latest/reference/lambda/invoke.html)
```
Comment (tinyg210) [inline content/en/user-guide/aws/lambda/index.md:68]:
```suggestion
Invoke the function directly. See the [Lambda CLI invoke docs](https://docs.aws.amazon.com/cli/latest/reference/lambda/invoke.html) for more information.
```

File: content/en/user-guide/aws/lambda/index.md [hunk 3]
```diff
@@ -64,6 +64,38 @@ $ awslocal lambda create-function \
...
 {{< /alert >}}
 
+### Invoke the Function with AWS CLI
+Invoke the function directly. See ['Lambda CLI invoke docs'](https://docs.aws.amazon.com/cli/latest/reference/lambda/invoke.html) 
+The command below encodes the body as a JSON string like an event coming from API Gateway or Lambda URL does.
+This requires AWS CLI v2. To see examples of using the 3 ways to use the AWS CLI with LocalStack, see this repo. [Simple AWS CLI Sample with Lambda](https://github.com/localstack-samples/sample-simple-invoke-lambda)
+
+
+#### Using awslocal
+This should be using AWS ClI v2 similar to this output.
```
Comment (tinyg210) [inline content/en/user-guide/aws/lambda/index.md:74]:
```suggestion
This should be using AWS ClI v2, similar to this output.
```

**Review by tinyg210 — APPROVED** [review]
I left some suggestions of nitpicks that would improve the reading experience. Other than that  I think there's no point in dragging this on forever.

**Comment by MarcelStranak** [comment]
Could we maybe add it as tabpanes like in [installation docs](https://docs.localstack.cloud/getting-started/installation)?
Use tabs like awslocal v1, aws cli v2? I am happy to give it a try if you are okay with it.

**Comment by cabeaulac** [comment]
> Could we maybe add it as tabpanes like in [installation docs](https://docs.localstack.cloud/getting-started/installation)?
Use tabs like awslocal v1, aws cli v2? I am happy to give it a try if you are okay with it.

Hi @MarcelStranak. I merged this PR. The current directions require AWS CLI v2. If you want to add v1 and use tabs, that is totally fine. I recommend doing it in another ticket. These directions updates were pending for too long. Thanks!

**Pull request merged**